plugins/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 add_subdirectory(codecs)
 add_subdirectory(filters)
 
+set(SOURCES ${SOURCES} PARENT_SCOPE)
```

Comment:
It is a good practice to always add an empty line at the end of a file.

Reply:
solved in d62e4d8